src/OpenTelemetry.Collector.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -34,8 +34,8 @@ internal class HttpInListener : ListenerHandler<HttpRequest>
         private readonly PropertyFetcher beforeActionTemplateFetcher = new PropertyFetcher("Template");
         private readonly bool hostingSupportsW3C = false;
 
-        public HttpInListener(ITracer tracer, Func<HttpRequest, ISampler> samplerFactory)
-            : base("Microsoft.AspNetCore", tracer, samplerFactory)
+        public HttpInListener(string name, Version version, ITracerFactory tracerFactory, Func<HttpRequest, ISampler> samplerFactory)
```

Comment:
this is a tracer specific to incoming HTTP requests from ASP.NET Core, I'd argue this should get tracer specific to it and never deal with factory.